client/src/inside/components/EhrBanner.vue

```diff
@@ -14,7 +14,7 @@
             li Gender: &nbsp;
               b {{ demographics.gender }}
             li Weight: &nbsp;
-              b 49kg
+              b {{ `${demographics.weight || 49}kg` }}
```

Comment:
So it was quite simple after all.  Please remove the default.  Let's just use the value from the demographics page.